docs/pages/workflow/customizing.mdx

```diff
@@ -5,13 +5,13 @@ description: Learn how to add custom native code to your Expo project.
 
 import { Terminal } from '~/ui/components/Snippet';
 
-Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a [development build](/workflow/overview/#development-builds) allows you to include your specific native dependencies and customizations and not what's in the latest [Expo SDK](/versions/latest/). These native customizations can be any React Native library or your own custom native code.
+Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a [development build](/workflow/overview/#development-builds) allows you to include your specific native dependencies and customizations and not only what comes with the latest [Expo SDK](/versions/latest/). These native customizations can be any React Native library or your own custom native code.
 
-For most third-party native libraries, [autolinking](/modules/autolinking/) makes it easy. All you have to do is install the package and create a new development build to use the module. Some modules might also need a [config-plugin](/config-plugins/introduction) for additional native customizations. When adding your own custom native code, the approach varies based on how you want to work with native code.
+For most third-party native libraries, [autolinking](/modules/autolinking/) makes it easy. All you have to do is install the npm package and run [prebuild](/workflow/prebuild/) to use the module. Some modules might also need a [config-plugin](/config-plugins/introduction) for additional native customizations. When adding your own custom native code, the approach varies based on how you want to work with native code.
 
 ## Create a local module with the Expo Modules API
 
-By default, Expo projects use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation). This means that projects do not have **android** and **ios** directories containing the native code and configuration. Instead, the native directories are generated right before building the app and is based on your [app config](/workflow/configuration) and native modules in **package.json**. This simplifies the management and upgrading of native code.
+By default, Expo projects use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation). This means that projects do not have **android** and **ios** directories containing the native code and configuration. Instead, the native directories are generated right before building the app and are based on your [app config](/workflow/configuration) and native modules in **package.json**. This greatly simplifies the management and upgrading of native code.
```

Comment:
```suggestion
By default, Expo projects use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation). This means that projects do not have **android** and **ios** directories containing the native code and configuration. Instead, the native directories are generated right before building the app and are based on your [app config](/workflow/configuration) and native modules in **package.json**. This simplifies the management and upgrading of native code.
```

Let's stick to using a more neutral tone in the last sentence. Whether the impact is large or not, we'll leave that for the user to decide. The impact can also depend on other factors, such as the scope of the user's project.